lib/model/vertex.js

```diff
@@ -62,8 +104,24 @@ var Vertex = createClass({
       initOnly: true
     },
 
+    /**
+     * Key/value pairs of styles
+     * Guaranteed to have default styles set
+     * (which come from the static DEFAULT_STYLES on Vertex)
+     * @attribute styles
+     * @type      Object
+     */
     styles: {
-      //dunno
+      validator: Lang.isObject,
+      valueFn: function() {
+        return Vertex.DEFAULT_STYLES;
+      },
+      
+      setter: function(val) {
+        // merge set styles w/ defaults
+        // so that it's guaranteed to have defaults set
+        return ObjectUtils.merge(Vertex.DEFAULT_STYLES, val);
+      }
```

Comment:
I was thinking more about this, and I'm not sure whether it makes more sense for the data model to hold a list of styles (like `"server"` or `"critical"`) or a style description (like `{border: '10px solid black; background: 'red'}`. This concern might be outside of the scope of this PR.